src/containers/Tenant/Diagnostics/Overview/Overview.tsx

```diff
@@ -122,6 +120,10 @@ function Overview({type, tenantName, className}: OverviewProps) {
                 }
                 dispatch(getOlapStats({path: schemaPath}));
             }
+
+            if (isPathTypeWithTopic(type)) {
+                dispatch(getTopic(currentSchemaPath));
```

Comment:
Topic data is requested here because of loading and isLoading flags. When these flags are reset, overview child components are mounted again, so topic data is loaded twice if it is fetched inside child components